scipy/stats/_continuous_distns.py

```diff
@@ -6015,6 +6015,12 @@ def _stats(self, df):
         g2 = np.where(df <= 2, np.nan, g2)
         return mu, mu2, g1, g2
 
+    def _entropy(self, df):
+        half = df/2
+        half1 = (df + 1)/2
+        return (half1*(sc.digamma(half1) - sc.digamma(half)) +
+                np.log(np.sqrt(df)*sc.beta(half, 0.5)))
```

Comment:
```suggestion
        return (half1*(sc.digamma(half1) - sc.digamma(half))
                + np.log(np.sqrt(df)*sc.beta(half, 0.5)))
```

Reply:
Committed.  I've always had a slight preference for the old style, but Knuth-style is the current recommendation, so +1.